assets/xml/objects/object_uch.xml

```diff
@@ -1,46 +1,64 @@
-        <Texture Name="object_uch_Tex_004180" OutName="tex_004180" Format="ci8" Width="16" Height="32" Offset="0x4180" />
-        <Texture Name="object_uch_Tex_004380" OutName="tex_004380" Format="ci8" Width="16" Height="32" Offset="0x4380" />
-        <Texture Name="object_uch_Tex_004580" OutName="tex_004580" Format="rgba16" Width="32" Height="32" Offset="0x4580" />
-        <Limb Name="object_uch_Standardlimb_004D80" Type="Standard" EnumName="OBJECT_UCH_LIMB_01" Offset="0x4D80" />
-        <Limb Name="object_uch_Standardlimb_004D8C" Type="Standard" EnumName="OBJECT_UCH_LIMB_02" Offset="0x4D8C" />
-        <Limb Name="object_uch_Standardlimb_004D98" Type="Standard" EnumName="OBJECT_UCH_LIMB_03" Offset="0x4D98" />
-        <Limb Name="object_uch_Standardlimb_004DA4" Type="Standard" EnumName="OBJECT_UCH_LIMB_04" Offset="0x4DA4" />
-        <Limb Name="object_uch_Standardlimb_004DB0" Type="Standard" EnumName="OBJECT_UCH_LIMB_05" Offset="0x4DB0" />
-        <Limb Name="object_uch_Standardlimb_004DBC" Type="Standard" EnumName="OBJECT_UCH_LIMB_06" Offset="0x4DBC" />
-        <Limb Name="object_uch_Standardlimb_004DC8" Type="Standard" EnumName="OBJECT_UCH_LIMB_07" Offset="0x4DC8" />
-        <Limb Name="object_uch_Standardlimb_004DD4" Type="Standard" EnumName="OBJECT_UCH_LIMB_08" Offset="0x4DD4" />
-        <Limb Name="object_uch_Standardlimb_004DE0" Type="Standard" EnumName="OBJECT_UCH_LIMB_09" Offset="0x4DE0" />
-        <Limb Name="object_uch_Standardlimb_004DEC" Type="Standard" EnumName="OBJECT_UCH_LIMB_0A" Offset="0x4DEC" />
-        <Limb Name="object_uch_Standardlimb_004DF8" Type="Standard" EnumName="OBJECT_UCH_LIMB_0B" Offset="0x4DF8" />
-        <Limb Name="object_uch_Standardlimb_004E04" Type="Standard" EnumName="OBJECT_UCH_LIMB_0C" Offset="0x4E04" />
-        <Limb Name="object_uch_Standardlimb_004E10" Type="Standard" EnumName="OBJECT_UCH_LIMB_0D" Offset="0x4E10" />
-        <Skeleton Name="object_uch_Skel_004E50" Type="Flex" LimbType="Standard" LimbNone="OBJECT_UCH_LIMB_NONE" LimbMax="OBJECT_UCH_LIMB_MAX" EnumName="object_uch_Limbs" Offset="0x4E50" />
+        <!-- Assets for the beam of light coming from the alien's eyes. -->
+        <DList Name="gAlienEyeBeamDL" Offset="0x80" />
+        <Texture Name="gAlienEyeBeamMaskTex" OutName="alien_eye_beam_mask" Format="i8" Width="32" Height="32" Offset="0x130" />
+        <TextureAnimation Name="gAlienEyeBeamTexAnim" Offset="0x550" />
+
+        <!-- An empty texture animation, seemingly planned to animate something on the alien's body. -->
+        <TextureAnimation Name="gAlienEmptyTexAnim" Offset="0x560" />
+
+        <!-- Alien Animations -->
+        <Animation Name="gAlienDeathAnim" Offset="0x608" /> <!-- Original name is "uch_die_body" -->
+        <Animation Name="gAlienDamageAnim" Offset="0x6C8" /> <!-- Original name is "uch_gaku_body" (probably short for the onomatopoeia for "jerking") -->
```

Comment:
Ideally we'd describe these by what they look like rather than how they're used.

Reply:
These two animations look almost identical (these are both animated GIFs):
![damage](https://user-images.githubusercontent.com/12245827/177375631-c72a9406-0f37-4227-ba72-8ba35ead7eda.gif)
![death](https://user-images.githubusercontent.com/12245827/177375643-d746ca1f-0a06-4d0a-baa5-087bfabdf0aa.gif)

So describing them based on how they look seems counterproductive.

(We've also used die/death and damage/damaged as animation names in the past without issue, so it'd be pretty weird/almost kind of funny to switch away from that for the object where they are almost completely identical).

Reply:
Talked it over in discord and I'm swapping to `gAlienJerkingAnim`. The death one is staying though.